### PR TITLE
Correctly pluralize "bytes", "patches", etc

### PR DESCRIPTION
**Pull request by Rangi42**

### CODE REVIEW

**Review by ISSOtm — CHANGES_REQUESTED** [review]
Perhaps this could be factored out into a

```rust
struct Plural(&'static str, usize);

impl Display for Plural {
    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
        if self.1 == 0 {
            Ok(())
        } else {
            f.write_str(self.0)
        }
    }
}
```

...and then `println!("{len} patch{}", Plural("es", len))`.

**Comment by Rangi42** [comment]
> Perhaps this could be factored out into a `struct Plural(&'static str, usize);` ...and then println!("{len} patch{}", Plural("es", len)).

This would require casting a couple of sites, e.g. `Plural("s", len as usize)`. It would also not be a fully-general pluralization support, e.g. `println!("{} loa{}", len, if len == 1 { "f" } else { "ves" })` wouldn't work. So I think it's on the borderline of being too abstract for the given problem.

File: src/main.rs [hunk 1]
```diff
@@ -318,7 +331,13 @@ fn work(args: &Args) -> Result<(), MainError> {
...
             if args.section.get(SectionFeatures::SIZE) {
                 let len = section.size();
-                println!("{}${:04x} ({}) bytes", indent, len, len);
+                println!(
+                    "{}${:04x} ({}) bytes{}",
+                    indent,
+                    len,
+                    len,
+                    plural!(len, "s")
+                );
```
Comment (ISSOtm) [inline src/main.rs:340]:
```suggestion
                println!(
                    "{indent}${len:04x} ({len}) bytes{}",
                    plural!(len, "s"),
                );
```
In the interest of compactness.

**Review by ISSOtm — DISMISSED** [review]

File: src/main.rs (hunk 1 above)
Comment (Rangi42) [reply src/main.rs:340]: There are lots of other `println("{}", somevar)`s; should I change them in general to `println!("{somevar}")`?

Comment (ISSOtm) [reply src/main.rs]: Please do!

**Review by ISSOtm — APPROVED** [review]
Thank you!

**Pull request merged**